setup.py

```diff
@@ -1,20 +1,17 @@
 from setuptools import setup
 
-readme = open('README.rst').read()
 history = open('CHANGES.rst').read().replace('.. :changelog:', '')
 
 setup(
-    name='sqlalchemy-redshift',
+    name='gx-sqlalchemy-redshift',
     version='0.8.15.dev0',
```

Comment:
I think we should change the version to something like 0.1.0?

Reply:
That's fine. I would like to indicate we are out of dev with respect to using with GX. This also isn't blocking.